Configuration.props

```diff
@@ -159,8 +159,8 @@
     <AndroidToolPath Condition=" '$(AndroidToolPath)' == '' ">$(AndroidSdkFullPath)\tools</AndroidToolPath>
     <AndroidToolsBinPath Condition=" '$(AndroidToolsBinPath)' == '' ">$(AndroidToolPath)\bin</AndroidToolsBinPath>
     <AndroidToolExe Condition=" '$(AndroidToolExe)' == '' ">android</AndroidToolExe>
-    <CommandLineToolsFolder Condition=" '$(CommandLineToolsFolder)' == '' ">1.0</CommandLineToolsFolder>
-    <CommandLineToolsVersion Condition=" '$(CommandLineToolsVersion)' == '' ">6200805_latest</CommandLineToolsVersion>
+    <CommandLineToolsFolder Condition=" '$(CommandLineToolsFolder)' == '' ">4.0</CommandLineToolsFolder>
```

Comment:
One piece I noticed recently was that we have a mismatch here and in the version the auto sdk tooling uses: https://github.com/xamarin/xamarin-android-tools/blob/26d65d95943b18bb9575d75c8c60ce36180a0db9/src/Xamarin.Android.Tools.AndroidSdk/Xamarin.Android.Tools.Versions.props#L13.  We can probably update xaprepare to read the folder version from xamarin-android-tools instead, but I think that can be done separately if we want.